graylog2-server/src/test/java/org/graylog2/rest/resources/system/contentpacks/CatalogResourceTest.java

```diff
@@ -139,9 +140,9 @@ public void getTitles() {
         );
         doReturn(expectedResponse)
                 .when(entityTitleService)
-                .getTitles(any(), eq(request));
+                .getTitles(eq(request), any());
```

Comment:
It is very minor, but I would replace `any()` with `eq(<test search user instance>)`